x-pack/plugins/actions/server/builtin_action_types/alert_history_es_index/alert_history_es_index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { PreConfiguredAction } from '../../types';
+import { ActionTypeId as EsIndexActionTypeId } from '../es_index';
+import { AlertHistoryEsIndexConnectorId, AlertHistoryDefaultIndexName } from '../../../common';
+
+export function getAlertHistoryEsIndex(): Readonly<PreConfiguredAction> {
+  return Object.freeze({
+    name: i18n.translate('xpack.actions.alertHistoryEsIndexConnector.name', {
+      defaultMessage: 'Alert History ES Index',
```

Comment:
@gchaps, should this name be in sentence case?

Reply:
Yes, Alert history ES index

Reply:
Updated in https://github.com/elastic/kibana/pull/94909/commits/07eb28d1aafe504de00a140ea51df04dfa8da50d